commit 41ed0738cc902c87629e6aefad1e40852508d5f8 (refs/changes/33/1233/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-01-11T21:29:23+00:00 (2 years, 9 months ago)
    
    MANTA-3078 muskie -> mako connection retries should be less aggressive
    Reviewed by: David Pacheco <dap@joyent.com>

diff --git a/main.js b/main.js
index 6957b95..b5a7606 100644
--- a/main.js
+++ b/main.js
@@ -225,7 +225,7 @@ function createCueballHttpAgent(cfg) {
                 retries: sharkCfg.retry.retries,
                 timeout: sharkCfg.connectTimeout,
                 maxTimeout: sharkCfg.maxTimeout,
-                delay: sharkCfg.maxTimeout
+                delay: sharkCfg.delay
             },
             'dns_srv': {
                 retries: 0,
diff --git a/sapi_manifests/muskie/template b/sapi_manifests/muskie/template
index 02b92a6..1273c31 100644
--- a/sapi_manifests/muskie/template
+++ b/sapi_manifests/muskie/template
@@ -98,10 +98,10 @@
   "sharkConfig": {
     {{! These are translated into cueball recovery parameters. }}
     "connectTimeout": 2000,
-    "maxTimeout": 10000,
-    "delay": 250,
+    "maxTimeout": 30000,
+    "delay": 500,
     "retry": {
-      "retries": 2
+      "retries": 5
     },
 
     {{! Use bootstrap mode to find binder, same as for cueballHttpAgent. }}
